### DISCUSSION TITLE
Storefront doesn't display Payment providers

### DISCUSSION THREAD

**Discussion by quang-ng** [saleor/saleor-platform#216] [Q&A]
Hi Community, 

After follow docs `Installation with Docker` (https://docs.saleor.io/docs/3.x/setup/docker-compose), everything work well but I have a trouble with setup payment using `saleor-platform`. It not display any Payment provider in Checkout page event I active Stripe plugin. There are some screenshots. Please help me, there is any configure that I missing??

![image](https://user-images.githubusercontent.com/6895574/207773684-79069fe2-a736-4077-a48e-1c97b2f3de05.png)

![image](https://user-images.githubusercontent.com/6895574/207773943-39388597-4776-4b88-bc91-9b1d7ece1365.png)

The commands I run to installation: 

```
git clone https://github.com/saleor/saleor-platform.git --recursive --jobs 3
cd saleor-platform
docker-compose build

docker-compose run --rm api python3 manage.py migrate
docker-compose run --rm api python3 manage.py collectstatic --noinput

docker-compose run --rm api python3 manage.py populatedb

docker-compose run --rm api python3 manage.py createsuperuser

docker-compose up
```

**Comment by varada13** [comment]
i also followed the same steps , for me client app is ot even running

**Comment by fedexin40** [comment]
@quang-ng do you find the error? I facing the same issue

**Comment by quang-ng** [comment]
Not yet @fedexin40. I moved to use Medusajs.
